Bottom: b5459e9c317707c0792bf02c210bafaa4a4031e9
Top:    6e79acc95e5cd324e988850c7bb7a5ba4b71e9ee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 14:33:38 +0100

Refresh of generate-pcb-menu-res-and-gpcb

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 21a9f5a..f57483c 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -18,7 +18,7 @@ EXTRA_LIBRARIES = \
 	liblpr.a libgerber.a libbom.a libpng.a libps.a libnelma.a \
 	libgcode.a
 
-pcblib_DATA=	\
+pcblib_DATA= \
 	default_font \
 	gpcb-menu.res \
 	pcb-menu.res
@@ -457,7 +457,9 @@ DISTCLEANFILES= pcbtest.sh gpcb-menu.h pcb-menu.h \
 	hid/nelma/nelma_lists.h \
 	hid/ps/ps_lists.h \
 	core_lists.h \
-	dbus-introspect.h
+	dbus-introspect.h \
+	gpcb-menu.res \
+	pcb-menu.res
 
 # create wrapper script that lets you test pcb prior to installation
 pcbtest.sh:	$(srcdir)/pcbtest.sh.in Makefile
